runtime/doc/eval.txt

```diff
@@ -4053,6 +4053,8 @@ jobstart({cmd}[, {opts}])				{Nvim} *jobstart()*
 		  - width: Width of the terminal screen(only if pty is set)
 		  - height: Height of the terminal screen(only if pty is set)
```

Comment:
can you fix these parenthese while you're here:

     - width: Width of the terminal screen (only if pty is set)
     - height: Height of the terminal screen (only if pty is set)